templates/cisco_nxos_show_interface_switchport.template

```diff
@@ -0,0 +1,14 @@
+Value Required INTERFACE (\S+)
+Value SWITCHPORT (\S+)
+Value MODE (\S+)
+Value ACCESS_VLAN (\S+)
+Value NATIVE_VLAN (\S+)
+Value TRUNKING_VLANS (\S+)
+
+Start
+  ^Name: ${INTERFACE}
+  ^\s*Switchport: ${SWITCHPORT}
+  ^\s*Operational Mode: ${MODE}
+  ^\s*Access Mode VLAN: ${ACCESS_VLAN}
+  ^\s*Trunking Native Mode VLAN: ${NATIVE_VLAN}
+  ^\s*Trunking VLANs Allowed: ${TRUNKING_VLANS} -> Record
```

Comment:
Do you mind changing from static spaces to using \s* or \s+ within the state?

Do you mind adding the following to the end of Start state:
```
  ^\s*$$
  ^. -> Error
```
Note: This may cause failures if not all lines of output are accounted for, which appears will be the case.

Also, it may be better to change **MODE** to **OPER_MODE** to match the IOS template.